compiler/rustc_infer/src/infer/mod.rs

```diff
@@ -184,6 +184,14 @@ impl<'tcx> InferCtxtInner<'tcx> {
         self.projection_cache.with_log(&mut self.undo_log)
     }
 
+    #[inline]
+    fn try_type_variables_probe_ref(
+        &self,
+        vid: ty::TyVid,
+    ) -> Option<&type_variable::TypeVariableValue<'tcx>> {
+        self.type_variable_storage.eq_relations_ref().try_probe_value(vid)
```

Comment:
Where is `try_probe_value` defined?

Reply:
`eq_relations_ref` returns a type from ena, so probably over there.

Reply:
I overlooked that there was a corresponding `ena` update, thanks.